scandium-core/src/test/java/org/eclipse/californium/scandium/dtls/AdversaryClientHandshaker.java

```diff
@@ -0,0 +1,239 @@
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *    Matthias Kovatsch - creator and main architect
+ *    Stefan Jucker - DTLS implementation
+ *    Kai Hudalla (Bosch Software Innovtions GmbH) - small improvements
+ *    Kai Hudalla (Bosch Software Innovations GmbH) - store peer's identity in session as a
+ *                                                    java.security.Principal (fix 464812)
+ *    Kai Hudalla (Bosch Software Innovations GmbH) - notify SessionListener about start and completion
+ *                                                    of handshake
+ *    Kai Hudalla (Bosch Software Innovations GmbH) - fix 475112: only prefer RawPublicKey from server
+ *                                                    if no trust store has been configured
+ *    Kai Hudalla (Bosch Software Innovations GmbH) - consolidate and fix record buffering and message re-assembly
+ *    Kai Hudalla (Bosch Software Innovations GmbH) - replace Handshaker's compressionMethod and cipherSuite
+ *                                                    properties with corresponding properties in DTLSSession
+ *    Kai Hudalla (Bosch Software Innovations GmbH) - derive max fragment length from network MTU
+ *    Kai Hudalla (Bosch Software Innovations GmbH) - use SessionListener to trigger sending of pending
+ *                                                    APPLICATION messages
+ *    Achim Kraus (Bosch Software Innovations GmbH) - use isSendRawKey also for
+ *                                                    supportedServerCertificateTypes
+ *    Ludwig Seitz (RISE SICS) - Updated calls to verifyCertificate() after refactoring
+ *    Bosch Software Innovations GmbH - migrate to SLF4J
+ *    Achim Kraus (Bosch Software Innovations GmbH) - add certificate types only,
+ *                                                    if certificates are used
+ *    Achim Kraus (Bosch Software Innovations GmbH) - issue #549
+ *                                                    trustStore := null, disable x.509
+ *                                                    trustStore := [], enable x.509, trust all
+ *    Achim Kraus (Bosch Software Innovations GmbH) - fix usage of literal address
+ *                                                    with enabled sni
+ *    Vikram (University of Rostock) - added ECDHE_PSK mode
+ *    Achim Kraus (Bosch Software Innovations GmbH) - add handshake parameter available to
+ *                                                    process reordered handshake messages
+ *    Achim Kraus (Bosch Software Innovations GmbH) - add dtls flight number
+ *    Achim Kraus (Bosch Software Innovations GmbH) - redesign DTLSFlight and RecordLayer
+ ******************************************************************************/
```

Comment:
is it intended to keep all this contributors in the header of this new class ?

Reply:
Uups, copy&paste